common/src/main/java/com/wynntils/wc/utils/scoreboard/ScoreboardManager.java

```diff
@@ -140,8 +155,53 @@ public class ScoreboardManager {
                 }
             }
         }
+
+        handleScoreboardReconstruction();
     };
 
+    private static void handleScoreboardReconstruction() {
+        Scoreboard scoreboard = McUtils.player().getScoreboard();
+
+        List<String> skipped = new ArrayList<>();
+
+        for (Segment parsedSegment : segments) {
+            boolean cancelled = WynntilsMod.getEventBus().post(new ScoreboardSegmentAdditionEvent(parsedSegment));
+
+            if (cancelled) {
+                skipped.addAll(parsedSegment.getScoreboardLines());
```

Comment:
I'm thinking that if all segments are removed, maybe the entire scoreboard should be hidden..?

Reply:
It is hidden that case.

Reply:
The reason you don't see code for that is, no matter if we have a display scoreboard, if there are no scores, Minecraft does not display it.

Reply:
What about the top heading "play.wynncraft.com"?

Reply:
That is the scoreboard name, if no "players" have score, it doesn't display.

Reply:
Ok, fine!